edx_exams/apps/lti/views.py

```diff
@@ -13,31 +13,53 @@
 from lti_consumer.api import get_end_assessment_return, get_lti_1p3_launch_start_url
 from lti_consumer.data import Lti1p3LaunchData, Lti1p3ProctoringLaunchData
 from lti_consumer.models import LtiConfiguration
-from rest_framework.decorators import authentication_classes, permission_classes
+from rest_framework import status
+from rest_framework.decorators import api_view, authentication_classes, permission_classes
 from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
 
+from edx_exams.apps.core.api import get_attempt_by_id, update_attempt_status
 from edx_exams.apps.core.models import ExamAttempt
+from edx_exams.apps.core.statuses import ExamAttemptStatus
 from edx_exams.apps.lti.utils import get_lti_root
 
 EDX_OAUTH_BACKEND = 'auth_backends.backends.EdXOAuth2'
 
 
+@api_view(['GET'])
 @require_http_methods(['GET'])
 @authentication_classes((JwtAuthentication,))
 @permission_classes((IsAuthenticated,))
 def start_proctoring(request, attempt_id):
     """
     LTI Start Proctoring
     """
-    # TODO: Here we'd do all the start of proctoring things.
+    # Verify that the attempt associated with attempt_id exists.
+    attempt = get_attempt_by_id(attempt_id)
+    if not attempt:
+        return Response(
```

Comment:
something to follow up with as a 'nice to have' ticket but these are going to end up rendered in the user's browser. We might want to return a nicer HTML fragment or something.

Reply:
That's a good point. I'll log a ticket.

Reply:
I logged [MST-1865](https://2u-internal.atlassian.net/browse/MST-1865).